package.json

```diff
@@ -149,6 +149,7 @@
     "css-loader": "^6.8.1",
     "dotenv": "^16.3.1",
     "eslint": "^8.50.0",
+    "eslint-config-prettier": "^9.0.0",
```

Comment:
Is this all we really need for this plugin to work? I would have expected some changes in this file, inside the existing `eslintConfig` attribute.